docs/checkit.js

```diff
@@ -460,6 +460,22 @@ CheckIt.prototype.generateCheckmarks = function(calObj, $div) {
      })
 };
 
+CheckIt.prototype.removeEmptyWeeks = function(calObj, $div) {
```

Comment:
`$div` could have a more descriptive name. This function can't take just any div. It has to be a `$calendarDiv` that contains `.month` and `.week` etc.